controller/work_item_link_type.go

```diff
@@ -44,24 +50,20 @@ func enrichLinkTypeSingle(ctx *workItemLinkContext, single *app.WorkItemLinkType
 	// Now include the optional link category data in the work item link type "included" array
 	modelCategory, err := ctx.Application.WorkItemLinkCategories().Load(ctx.Context, single.Data.Relationships.LinkCategory.Data.ID)
 	if err != nil {
-		jerrors, httpStatusCode := jsonapi.ErrorToJSONAPIErrors(err)
-		return ctx.ResponseData.Service.Send(ctx.Context, httpStatusCode, jerrors)
-
+		return err
 	}
 	appCategory := convertLinkCategoryFromModel(*modelCategory)
 	single.Included = append(single.Included, appCategory.Data)
 
 	// Now include the optional link space data in the work item link type "included" array
 	space, err := ctx.Application.Spaces().Load(ctx.Context, *single.Data.Relationships.Space.Data.ID)
 	if err != nil {
-		jerrors, httpStatusCode := jsonapi.ErrorToJSONAPIErrors(err)
-		return ctx.ResponseData.Service.Send(ctx.Context, httpStatusCode, jerrors)
+		return err
 	}
 
 	spaceData, err := ConvertSpaceFromModel(ctx.Context, ctx.DB, ctx.RequestData, *space)
 	if err != nil {
-		jerrors, httpStatusCode := jsonapi.ErrorToJSONAPIErrors(err)
-		return ctx.ResponseData.Service.Send(ctx.Context, httpStatusCode, jerrors)
+		return err
```

Comment:
@aslakknutsen I responded to @hectorj2f's comment above for an explanation. Please let me know if this is a correct change.